SYCL/Basic/accessor/accessor.cpp

```diff
@@ -827,6 +878,73 @@ int main() {
     }
   }
 
+  // Test byte_size(), size(), max_size(), empty()
+  {
+    std::vector<char> vecChar(64, 'a');
+    std::vector<int> vecInt(32, 1);
+    std::vector<float> vecFloat(16, 1.0);
+    std::vector<IdxID3> vecCustom(8, {0, 0, 0});
+    TestAccSizeFuncs(vecChar);
+    TestAccSizeFuncs(vecInt);
+    TestAccSizeFuncs(vecFloat);
+    TestAccSizeFuncs(vecCustom);
+  }
+  // Test swap() on host_accessor
+  {
+    std::vector<int> vec1(8), vec2(16);
+    try {
+      sycl::buffer<int> buf1(vec1.data(), vec1.size());
+      sycl::buffer<int> buf2(vec2.data(), vec2.size());
+      auto acc1 = buf1.get_access<sycl::access::mode::read_write>();
+      auto acc2 = buf2.get_access<sycl::access::mode::read_write>();
+      acc1.swap(acc2);
+      acc1[15] = acc2[7] = 4;
```

Comment:
IMO, this is too smart. Can we split it into two statements?

Reply:
Okay, [db9dbf3](https://github.com/intel/llvm-test-suite/pull/1265/commits/db9dbf3ce588ab34ddae003b0dc32de5c58d1aa5)